opensearch/src/main/java/org/opensearch/sql/opensearch/storage/scan/OpenSearchIndexScanQueryBuilder.java

```diff
@@ -99,6 +100,13 @@ public boolean pushDownHighlight(LogicalHighlight highlight) {
     return true;
   }
 
+  @Override
+  public boolean pushDownNested(LogicalNested nested) {
+    indexScan.getRequestBuilder().pushDownNested(nested.getFields());
+    // Return false intentionally to keep the original nested operator
+    return false;
```

Comment:
To keep the PhysicalPlan for `UnnestOperator` to iterate over the input from OS. I am having issues with this currently because I want the `ProjectOperator` pushed down while also keeping the PhysicalPlan for the `UnnestOperator` to do processing on the input from OS. After pushDown if we return true then that portion of the  PhysicalPlan is removed from the list.